bucket_test.go

```diff
@@ -464,6 +464,217 @@ func TestBucket_Delete_NonExisting(t *testing.T) {
 	}
 }
 
+func TestBucket_MoveSubBucket(t *testing.T) {
+	testCases := []struct {
+		name                    string
+		srcBucketName           string
+		subBucketName           string
+		subBucketKey            string
```

Comment:
Maybe we can add a case to create non-inline sub bucket here.

Reply:
u mean to use the src bucket as `root` bucket ?

Reply:
For the bucket, if the key or sub-bucket is small, the bucket can store those in value. It' s inline type.
If the key/value is large enough to allocate new page(s) for that, the bucket value only stores the root page id.
Just want to verify these two cases.

Yeah, since you change the `tx.MoveSubBucket` to support root bucket, it should add some cases to cover that.